content/en/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -25,12 +25,10 @@ cluster. This can help to reduce costs and improve performance.
 
 ## Using Service Internal Traffic Policy
 
-The `ServiceInternalTrafficPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-is a Beta feature and enabled by default.
-When the feature is enabled, you can enable the internal-only traffic policy for a
-{{< glossary_tooltip text="Service" term_id="service" >}}, by setting its
-`.spec.internalTrafficPolicy` to `Local`.
-This tells kube-proxy to only use node local endpoints for cluster internal traffic.
+The `ServiceInternalTrafficPolicy` feature (GA in v1.26) enables the internal-only
+traffic policy for a {{< glossary_tooltip text="Service" term_id="service" >}}, by setting its
+`.spec.internalTrafficPolicy` to `Local`. This tells kube-proxy to only use node local
+endpoints for cluster internal traffic.
```

Comment:
```suggestion
You can enable the internal-only
traffic policy for a {{< glossary_tooltip text="Service" term_id="service" >}}, by setting its
`.spec.internalTrafficPolicy` to `Local`. This tells kube-proxy to only use node local
endpoints for cluster internal traffic.
```

We document stable features as if the text is being read several releases hence. See https://developers.google.com/style/timeless-documentation for some similar thinking.

The feature state shortcode is there to serve as a big clue that previous versions might not have included it.

Reply:
Thanks, used your suggestion